src/_playground/App.scss

```diff
@@ -84,6 +84,27 @@ $menu-spacing: 20px;
             height: 2rem;
         }
     }
+
+    &__screen-reader-only-focusable {
```

Comment:
I don't think it's necessary to make this a child class name concatenation.  I think this class could simply stand on its own.  Also, to follow BEM, I would make this class name `screen-reader-only--focusable`.

Reply:
@greg-a-smith just for clarification - you do not think screen-reader-only--focusable should be under frDocs-Menu?  just className: screen-reader-only--focusable?

Reply:
I actually meant that `screen-reader-only--focusable` could live at the root level (without any parent classes) since there's nothing specific about its styles related to the menu.